dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.auditCollector-1.0.jakarta.feature

```diff
@@ -0,0 +1,15 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName = io.openliberty.auditCollector-1.0.jakarta
```

Comment:
Seems convention is to remove the '-' in auto features. Applies to the name as well.
```suggestion
symbolicName = io.openliberty.auditCollector1.0.jakarta
```